### PR TITLE
FACT-1468 added docstrings to bulk-scan-processor

### PR DESCRIPTION
**Pull request by justiceia**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FACT-1468

### Change description ###
added docstrings

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/ErrorNotificationSender.java [hunk 1]
```diff
@@ -33,6 +41,14 @@ public ErrorNotificationSender(
         this.containerMappings = containerMappings;
     }
 
+    /**
+     * Sends an error notification to the error notification queue.
+     * @param zipFilename The zip file name
+     * @param containerName The container name
+     * @param eventId The event ID
+     * @param errorCode The error code
+     * @param cause The cause of the error
```
Comment (RuthKirby) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/ErrorNotificationSender.java:50]: Everything is in method signature order apart from the EnvelopeRejectionException cause param

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/jms/JmsErrorNotificationSender.java [hunk 2]
```diff
@@ -32,6 +40,14 @@ public JmsErrorNotificationSender(
         this.containerMappings = containerMappings;
     }
 
+    /**
+     * Sends an error notification to the error notifications queue.
+     * @param zipFilename The name of the zip file
+     * @param containerName The name of the container
+     * @param eventId The event ID
+     * @param errorCode The error code
+     * @param cause The exception that caused the error
```
Comment (RuthKirby) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/jms/JmsErrorNotificationSender.java:49]: Everything is in method signature order apart from the cause

**Review by RuthKirby — APPROVED** [review]
Looks good to me! :) A couple places of missing params or not in order but not that important.

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/jms/JmsErrorNotificationSender.java (hunk 2 above)
Comment (justiceia) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/jms/JmsErrorNotificationSender.java:49]: fixed

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/ErrorNotificationSender.java (hunk 1 above)
Comment (justiceia) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/ErrorNotificationSender.java:50]: fixed

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/OcrDataParseException.java [hunk 3]
```diff
@@ -13,6 +13,12 @@
  */
 public class OcrDataParseException extends JsonMappingException {
 
+    /**
+     * Creates a new instance of the exception.
+     *
+     * @param jsonParser the parser that was used to read the JSON content
+     * @param message the error message
+     */
```
[the remark this answers is not in the document]
Comment (justiceia) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/OcrDataParseException.java:22]: fixed

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/EnvelopeRejectingException.java [hunk 4]
```diff
@@ -1,6 +1,13 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.exceptions;
 
+/**
+ * This exception is thrown when envelope is processed in CCD.
+ */
 public class EnvelopeRejectingException extends RuntimeException {
+    /**
+     * Creates a new instance of the exception.
+     * @param message the exception message
+     */
```
[the remark this answers is not in the document]
Comment (justiceia) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/EnvelopeRejectingException.java:11]: fixed

**Pull request merged**